src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,10 +77,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    
+    //here a regular expression is used to search for an image url entered by the user
```

Comment:
nit: I think the proper way to do multi-line comments is /* .... */

Reply:
Oops, I'll fix this.